test/jdk/sun/net/www/http/HttpClient/KeepAliveTest.java

```diff
@@ -0,0 +1,590 @@
+            keepAliveKeyClassconstructor.setAccessible(true);
+            Object expectedKeepAliveKey = keepAliveKeyClassconstructor.newInstance(httpUrlConnection.getURL(), null);
+            System.out.println("ExpectedKeepAliveKey=" + expectedKeepAliveKey);
+            Object clientVectorObjectInMap = keepAliveCache.get(expectedKeepAliveKey);
+            System.out.println("ClientVector=" + clientVectorObjectInMap);
+            HttpClient httpClientCached = keepAliveCache.get(httpUrlConnection.getURL(), null);
+            System.out.println("HttpClient in Cache:" + httpClientCached);
+            if(httpClientCached != null) {
+                System.out.println("KeepingAlive:" + httpClientCached.isKeepingAlive());
+                System.out.println("UsingProxy:" + httpClientCached.getUsingProxy());
+                System.out.println("ProxiedHost:" + httpClientCached.getProxyHostUsed());
+                System.out.println("ProxiedPort:" + httpClientCached.getProxyPortUsed());
+                System.out.println("ProxyPortUsingSystemProperty:" + System.getProperty("http.proxyPort"));
+                System.out.println("ProxyHostUsingSystemProperty:" + System.getProperty("http.proxyHost"));
+                System.out.println("http.keepAlive.time.server=" + System.getProperty("http.keepAlive.time.server"));
+                System.out.println("http.keepAlive.time.proxy=" + System.getProperty("http.keepAlive.time.proxy"));
+
+                Class clientVectorClass = Class.forName("sun.net.www.http.ClientVector");
+                //      System.out.println("clientVectorClass=" + clientVectorClass);
+                Field napField = clientVectorClass.getDeclaredField("nap");
+                napField.setAccessible(true);
+                int napValue = (int) napField.get(clientVectorObjectInMap);
+                int actualValue = napValue / 1000;
+                //      System.out.println("nap=" + napValue / 1000);
+                System.out.printf("ExpectedOutput:%d ActualOutput:%d ", expectedValues[scenarioNumber], actualValue);
+                System.out.println();
+                if (expectedValues[scenarioNumber] != actualValue) {
+                    throw new RuntimeException(
+                            "ExpectedOutput:" + expectedValues[scenarioNumber] + " ActualOutput: " + actualValue);
+                }
+            } else {
+                //executed when value is not cached.
+                String expected = expectedValues[scenarioNumber] == 0 ? NOT_CACHED
+                    : String.valueOf(expectedValues[scenarioNumber]);
+                System.out.println("ExpectedOutput:" + expected + " ActualOutput:" + NOT_CACHED);
+                if (!expected.equalsIgnoreCase(NOT_CACHED)) {
+                    throw new RuntimeException("ExpectedOutput:" + expected + " ActualOutput:" + NOT_CACHED);
+                }
+            }
+        }
```

Comment:
There is an alignment problem here: it looks as if a close brace is missing,

Reply:
aligned properly. thanks